README.md

```diff
@@ -13,6 +13,7 @@ Twilio's Voice SDK allows you to add real-time voice and PSTN calling to your we
 * [API Docs](https://twilio.github.io/twilio-voice.js/index.html)
 * [More Docs](https://www.twilio.com/docs/voice/sdks/javascript)
 * [Quickstart](https://www.twilio.com/docs/voice/client/javascript/quickstart)
+* [Reference Components](https://github.com/twilio/twilio-voice-js-reference-components)
```

Comment:
Let's use the twilio docs https://www.twilio.com/docs/voice/sdks/javascript/reference-components